src/aiida_quantumespresso/workflows/functions/get_xspectra_structures.py

```diff
@@ -0,0 +1,325 @@
+                ' less than 0.'
+            )
+        elif supercell_min_parameter == 0:  # useful if no core-hole treatment is required
+            scale_unit_cell = False
+        else:
+            scale_unit_cell = True
+    else:
+        supercell_min_parameter = 8.0  # angstroms
+        scale_unit_cell = True
+    if 'standardize_structure' in unwrapped_kwargs.keys():
+        standardize_structure = unwrapped_kwargs['standardize_structure'].value
+        unwrapped_kwargs.pop('standardize_structure')
+    else:
+        standardize_structure = True
+    if 'absorbing_elements_list' in unwrapped_kwargs.keys():
+        elements_defined = True
+        abs_elements_list = unwrapped_kwargs['absorbing_elements_list'].get_list()
+        # confirm that the elements requested are actually in the input structure
+        for req_element in abs_elements_list:
+            if req_element not in elements_present:
+                raise ValidationError(
+                    f'Requested elements {abs_elements_list} do not match those in the given structure:'
+                    f' {elements_present}'
+                )
+    else:
+        elements_defined = False
+        abs_elements_list = [Kind.symbol for Kind in structure.kinds]
+    if 'is_molecule_input' in unwrapped_kwargs.keys():
+        is_molecule_input = unwrapped_kwargs['is_molecule_input'].value
+        # If we're working with a molecule, check for pymatgen_settings
+        if 'pymatgen_settings' in unwrapped_kwargs.keys():
+            pymatgen_settings_dict = unwrapped_kwargs.keys()['pymatgen_settings'].get_dict()
+            valid_keys = ['tolerance', 'eigen_tolerance', 'matrix_tolerance']
+            pymatgen_kwargs = {key: value for key, value in pymatgen_settings_dict.items() if key in valid_keys}
+        else:
+            pymatgen_kwargs = {}
+
+    if 'spglib_settings' in unwrapped_kwargs.keys():
+        spglib_settings_dict = unwrapped_kwargs['spglib_settings'].get_dict()
+        valid_keys = ['symprec', 'angle_tolerance']
+        spglib_kwargs = {key: value for key, value in spglib_settings_dict.items() if key in valid_keys}
+    else:
+        spglib_kwargs = {}
+
+    output_params = {}
+    result = {}
+
+    output_params['absorbing_elements_list'] = abs_elements_list
+
+    incoming_structure_size = len(structure.sites)
+    incoming_structure_cell = structure.cell
+    incoming_structure_params = structure.cell_lengths
+
+    output_params['input_structure_num_sites'] = incoming_structure_size
+    output_params['input_structure_cell_matrix'] = incoming_structure_cell
+    output_params['input_structure_cell_lengths'] = incoming_structure_params
+
+    # Process a non-periodic system
+    if is_molecule_input:
+        from pymatgen.symmetry.analyzer import PointGroupAnalyzer
+
+        pymatgen_molecule = structure.get_pymatgen_molecule()
+        centred_molecule = pymatgen_molecule.get_centered_molecule()
```

Comment:
```suggestion
        centered_molecule = pymatgen_molecule.get_centered_molecule()
```
Please change all of them.

Reply:
Ah, yes, apologies about those. That was me using the British English spelling, but I will change the spelling over for the sake of consistency.